templates/opps-list-widget.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * This template is used to display the list of flexible volunteer opportunities
+ * for the widget.
+ *
+ * To adjust this template copy it into your current theme within a folder called "wivm".
+ */
+
+// Global variables that are set in widget/class-widget.php via the widget settings in admin 
+global $display_opp_excerpt;
+global $display_opp_when;
+global $display_opp_where;
+global $display_opp_spots;
+
+$opp = new WI_Volunteer_Management_Opportunity( $post->ID ); //Get volunteer opportunity information
+
+?>
+
+<li>
+ 
+ <?php
+
+   // Display title of opportunity
+   the_title( sprintf( '<a href="%s" rel="bookmark">', esc_url( get_permalink() ) ), '</a>' );
```

Comment:
How about wrapping this in a h3 or h4 tag so it stands out a little from the meta information below? I definitely want to make sure there is some visual hierarchy.